File: ex03/ex03_02/ex03_02.js
import promptSync from 'prompt-sync';
const prompt = promptSync();

function main() {
  let x = 0;
  let y = 0;

  x = parseInt(prompt("Enter x: "));
  y = parseInt(prompt("Enter y: "));

  console.log(String(x) + " is greater than " +  String(y) + " is " + String(x > y));
  console.log(String(x) + " is less than " +  String(y) + " is " + String(x < y));
  console.log(String(x) + " is equal to " +  String(y) + " is " + String(x == y));

  return 0;
}

main();

File: ex03/ex03_02/ex03_02.py
def main():
  x = int(input("Enter x: "))
  y = int(input("Enter y: "))

  print(str(x) + " is greater than " + str(y) + " is " + str(x > y))
  print(str(x) + " is less than " + str(y) + " is " + str(x < y))
  print(str(x) + " is equal to " + str(y) + " is " + str(x == y))

if __name__ == "__main__":
  main()